package.json

```diff
@@ -46,6 +46,10 @@
     "through2": "^2.0.0",
     "uglify-js": "^2.4.16"
   },
+  "devEngines": {
+    "node": "0.12 || 4.x || 5.x || 6.x || 7.x",
```

Comment:
We support 0.10 on 6.x as well (circle ci) https://github.com/babel/babel/blob/master/circle.yml which we removed in 7